src/sentry/api/serializers/models/plugin.py

```diff
@@ -57,6 +57,7 @@ def serialize_field(project, plugin, field):
         'choices': field.get('choices'),
         'readonly': field.get('readonly', False),
         'defaultValue': field.get('default'),
+        'value': None
```

Comment:
this was just so that there wouldn't be undefined/null mismatch false positives when using `_.isEqual`